src/elfpy/utils/sim_utils.py

```diff
@@ -112,34 +112,38 @@ def get_init_lp_agent(
         Agent class that will perform the lp initialization action
     """
     # Wrapper functions are expected to have a lot of arguments
-    # pylint: disable=too-many-arguments
-    # get the reserve amounts for a small target liquidity to achieve a target pool APR
+    # pylint: disable=too-many-locals
```

Comment:
don't think we should disable this.  refactor instead.

Reply:
brb creating local_variables dataclass

I guess the idea is this is a hint that the function is doing too much, right? How could we split it up? I was thinking of putting the iteration into a function, but I don't think that actually eliminates any local variables.

I know. make them all global 🙌

Reply:
Dylan resolved this by combining calculations to avoid creating variables